src/Servant/Pagination.hs

```diff
@@ -407,6 +410,20 @@ instance ToHttpApiData (ContentRange fields res) where
     Text.pack (symbolVal field) <> " " <> (encodeText . toUrlPiece) start <> ".." <> (encodeText . toUrlPiece) end
 
 
+instance
+  ( HasPagination resource field
+  , IsRangeType (RangeType resource field)
+  ) => FromHttpApiData (ContentRange (field ': fields) resource) where
+  parseUrlPiece ""  = Left "Invalid Content Range"
```

Comment:
Why is a separate case for empty string needed? Looks like it would be handled by the case below.

Reply:
you are right, this is forgotten piece of code; before tried with another function which is partial